### ISSUE TITLE
Utility to fetch CSA headers from DICOM?

### ISSUE THREAD

**Issue by matthew-brett** [open-dicom/csa_header#16]
Nibabel has a `get_csa_header` function : https://github.com/nipy/nibabel/blob/master/nibabel/nicom/csareader.py#L31

It implements the logic of finding the `image` or `series` CSA header from the DICOM file.   I see from that file that there is logic for finding where the relevant private elements start: https://github.com/nipy/nibabel/blob/master/nibabel/nicom/utils.py#L7. .  So, for example, instead of doing this:

```python
raw_csa = dcm.get((0x29, 0x1020)).value
CsaHeader(raw_csa).read()
```

you can do this:

```
get_csa_header(dcm, 'image')
```

**Comment by matthew-brett** [comment]
I guess that could be implemented as:

```python
CSAHeader.from_dcm(dcm, 'image')
```

**Comment by ZviBaratz** [comment]
Absolutely, that would be fantastic.

**Comment by ZviBaratz** [comment]
Absolutely, that would be fantastic.

**Comment by matthew-brett** [comment]
What repository should we use to hold the example DICOM files for testing?

**Comment by matthew-brett** [comment]
E.g. for Nipraxis : https://github.com/nipraxis/nipraxis-data

**Comment by ZviBaratz** [comment]
I think it would be ideal if we could leverage [multiple download URLs](https://www.fatiando.org/pooch/latest/multiple-urls.html#multiple-download-urls) to integrate data from `dicom-gallery` type repositories. WDYT?

**Comment by matthew-brett** [comment]
Yup - looks to be trivial using the `urls` kwarg for Pooch : https://www.fatiando.org/pooch/v1.2.0/intermediate.html

**Comment by matthew-brett** [comment]
I'll refactor the Nipraxis fetcher into its own mini-package.  I've already refactored it so it uses a YaML config file to define its parameters.   So we can probably just drop the package into `csa_header`, in a bit.

**Comment by matthew-brett** [comment]
Refactored code into https://github.com/matthew-brett/unscrewed .  Will add mini data repository and example fetches from the DICOM gallery tomorrow.

**Comment by ZviBaratz** [comment]
Created a PR for a minimalistic integration with just Pooch. I think we might better leverage `unscrewed` when and if we integrate more testing data.
